deploy_staging.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+
+docker build -t gcr.io/${PROJECT_NAME_STG}/${DOCKER_IMAGE_NAME}:$TRAVIS_COMMIT ./docker
+
+echo $GCLOUD_SERVICE | base64 --decode -i > ${HOME}/gcloud-service-key.json
+cat ${HOME}/gcloud-service-key.json
+gcloud auth activate-service-account --key-file ${HOME}/gcloud-service-key.json
+
+gcloud --quiet config set project $PROJECT_NAME_STG
+gcloud --quiet config set container/cluster $CLUSTER_NAME_STG
+gcloud --quiet config set compute/zone ${CLOUDSDK_COMPUTE_ZONE}
+gcloud --quiet container clusters get-credentials $CLUSTER_NAME_STG
+
+gcloud docker -- push gcr.io/${PROJECT_NAME_STG}/${DOCKER_IMAGE_NAME}
```

Comment:
@nikhilrayaprolu yes something like that ... See

- https://github.com/fossasia/open-event-orga-server/blob/nextgen/kubernetes/travis/deploy-nextgen.sh#L33
- https://github.com/fossasia/open-event-orga-server/blob/nextgen/kubernetes/yamls/api-server/api-deployment.yml#L21
- https://hub.docker.com/r/eventyay/nextgen-api-server/tags/